masc/main.py

```diff
@@ -16,6 +14,8 @@
 from masc.dictionary import Dictionary
 from masc.masc_utils import MascUtils
 
+sys.path.extend(['.', '..'])
```

Comment:
I'm worried about this being before all the `masc` imports for a reason, but IMHO it looks like not very good practice to extend the system path within the script - looks like issues with how the software is packaged and can even be dangerous.

However running masc locally after the change apparently works, so I'm not super concerned :)

Reply:
It is there on the first place to be able to execute `python3 masc/main.py` or `python3 main.py` (it was the only way i managed to do it), as after the packaging refactor we could only execute the script (after cloning the repo, without installing the package) via `python3 -m masc.main`.

I see that it has been moved to after the masc imports, but i think it defeats the purpose of the statement being there in the first place, as it does not prevent the `ModuleNotFound` error anymore:

![image](https://user-images.githubusercontent.com/45886240/96338581-8143bd00-108f-11eb-8d25-1f3179aa60e7.png)

With that said, I agree with your statements, and i think the better option would be to get rid of it.

_If we don't, we may want to ignore those E402 warnings with a `# noqa: E402` comment._

Reply:
I think using `sys.path.extend(['.', '..'])` is a hack and the recommended way to develop should be using `pip install -e .` from the root. I will do changes to make this work and update the docs.

Reply:
@ibLeDy I have removed the path mangling and created https://github.com/sfaci/masc/issues/42 to update the recommended ways to install and develop masc.

Reply:
Indeed, i agree.